docs/pages/project/contributing/contributing-model-relationships.md

```diff
@@ -330,6 +329,8 @@ Each policy has a set of evaluation rules defined and the `evaluationQuery` attr
 1. To configure a relationship to be applied across all versions of a particular model, ensure the `version` property for those relationships is set to `*`, to limit the relationships to a specific version of a model, specify the correct model version.
 1. Specify `version` property as a regex to ensure relationships are applied to a subset of versions of a model.
 
+{% include alert.html title="Caution While Defining Relationships" type="warning" content="Relationships are not imported directly and they have to be packaged inside a model. Use caution when defining relationships that span models." %}
```

Comment:
```suggestion
{% include alert.html title="Gratuitous Scoping of Relationships" type="warning" content="Relationships are not imported directly and they have to be packaged inside a model. Use caution when defining relationships that span models." %}
```

Reply:
"Relationships are not imported directly and they have to be packaged inside a model." There is some relevance of this information to the warning, but it's not clear what that is given this phrasing and lack of explanation. Please revisit.

Reply:
"Use caution when defining relationships that span models." Great. Why? Where's the explanation here?

Reply:
This feedback has not been addressed.